.github/workflows/test-full.yml

```diff
@@ -1,12 +1,16 @@
 name: Branch
 
-# main/dev branches will get the full run across all OS/browsers
+# main/dev/release-* branches and nightly tags will get the full run across
+# all OS/browsers for multiple node versions
```

Comment:
This full workflow now runs on main/dev/release/nightly and runs for both node 18/20